engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-subscriptions.html

```diff
@@ -1,21 +1,24 @@
 <span page-title ng-model="activationKey">{{ 'Subscriptions for Activation Key:' | translate }} {{ activationKey.name }}</span>
 
 <section>
-  <h4 translate>Activation Key Type:</h4>
```

Comment:
I'm not sure what was meant by 'activation key type' so I made it consistent with the Content Host Details tab instead